crates/runc/src/lib.rs

```diff
@@ -945,3 +986,38 @@ mod tests {
         assert!(!response.output.is_empty());
     }
 }
+
+#[derive(Clone)]
+pub struct DefaultExecutor {}
```

Comment:
All custom executors will depend on Tokio anyway, so this can be omitted (see comments above).

Reply:
please refer to the answer of the first comment.